File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/Model/WorkingDay.cs
using System;
using HospitalLibrary.Core.Model.Enums;

namespace HospitalLibrary.Core.Model
{
    public class WorkingDay : BaseModel
    {
        public int DayOfWeek { get; set; }
        public Day Day { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public virtual Person User { get; set; }
    }
}

File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/workingDay.ts
import { Day } from "./day";

export class WorkingDay {
    id: number = 0;
    deleted: boolean = false;
    dayOfWeek: number = 0 ;
    day: Day = 0;
    startTime: Date = new Date();
    endTime: Date = new Date();

    public constructor(obj?: any) {
        if (obj) {
            this.id = obj.id;
            this.deleted = obj.deleted;
            this.dayOfWeek = obj.dayOfWeek;
            this.day = obj.day;
            this.startTime = obj.startTime;
            this.endTime = obj.endTime;        
        }
    }
}